src/components/Input/Input.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styles from "./Input.module.scss";
+
+export const Input = React.forwardRef(
+  ({ label, errorMessage, ...rest }, ref) => (
+    <label>
+      <div>{label}</div>
+      <input
+        ref={ref}
+        {...rest}
+        className={errorMessage ? styles["error-outline"] : ""}
+      />
+      <div className={styles["error-message"]}>{errorMessage}</div>
+    </label>
+  ),
+);
+
+export const Textarea = React.forwardRef(
```

Comment:
Maybe it would be worth it to split it into 2 separate files per component?

Maybe we could add storybook to Textarea as well?

Reply:
These components are pretty similar and have same logic. I would keep them in common file. Adding story for textarea

Reply:
Make sense - let's leave it that way 😉